### PR TITLE
feat(aps): add port information for vnc connections

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx [hunk 1]
```diff
@@ -26,6 +26,12 @@ The VNC protocol is platform-independent, supporting Linux, Windows, and macOS,
 - A [Mac mini](/bare-metal/apple-silicon/how-to/create-mac-mini/)
 
 ## Accessing the remote desktop
+
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```
Comment (nerda-codes) [inline bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx:31]: I think something is missing after "not" in this sentence "and does not the default VNC port 5900"

File: bare-metal/apple-silicon/quickstart.mdx [hunk 2]
```diff
@@ -51,6 +51,11 @@ You can access your Mac mini using VNC, a remote desktop-sharing protocol. It al
 
 Refer to our detailed documentation for Windows, Linux, and macOS for OS specific information: [How to access the remote desktop of a Mac mini](/bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini/).
 
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```
Comment (nerda-codes) [inline bare-metal/apple-silicon/quickstart.mdx:55]: same comment as above

File: bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx [hunk 3]
```diff
@@ -47,13 +47,18 @@ round-trip min/avg/max/stddev = 0.182/0.207/0.261/0.028 ms
 
 ## 2. Verify the VNC connection
 
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```
Comment (nerda-codes) [inline bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx:51]: same comment as above

File: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx (hunk 1 above)
Comment (bene2k1) [reply bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx:31]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```

File: bare-metal/apple-silicon/quickstart.mdx (hunk 2 above)
Comment (bene2k1) [reply bare-metal/apple-silicon/quickstart.mdx:55]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```

File: bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx (hunk 3 above)
Comment (bene2k1) [reply bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx:51]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```

**Review by nerda-codes — APPROVED** [review]

File: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx [hunk 4]
```diff
@@ -26,6 +26,12 @@ The VNC protocol is platform-independent, supporting Linux, Windows, and macOS,
 - A [Mac mini](/bare-metal/apple-silicon/how-to/create-mac-mini/)
 
 ## Accessing the remote desktop
+
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx:31]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```

File: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx [hunk 5]
```diff
@@ -26,6 +26,12 @@ The VNC protocol is platform-independent, supporting Linux, Windows, and macOS,
...
+  For manual connections, ensure that the port from the "Overview" section is correctly entered in your VNC client settings, as this is necessary for a successful connection.
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx:32]:
```suggestion
  For manual connections, ensure that the port from the **Overview** section is correctly entered in your VNC client settings, as this is necessary for a successful connection.
```

File: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx [hunk 6]
```diff
@@ -72,7 +78,7 @@ The VNC protocol is platform-independent, supporting Linux, Windows, and macOS,
           - Open Remote Desktop Manager
           - Click **+** and **New Entry**
           - Choose **ARD**
-          - Enter the IP address of the Mac mini in the "Host" field, the username, and the password (found in the console)
+          - Enter the Mac mini's IP address in the "Host" field. Then, specify the port in the "Port" field, along with your username and password (located in the console).
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx:81]:
```suggestion
          - Enter the Mac mini's IP address in the **Host** field. Then, specify the port in the **Port** field, along with your username and password (located in the console).
```

File: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx [hunk 7]
```diff
@@ -107,7 +113,7 @@ The VNC protocol is platform-independent, supporting Linux, Windows, and macOS,
...
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx:116]:
```suggestion
          - Enter the Mac mini's IP address in the **Host** field. Then, specify the port in the **Port** field, along with your username and password (located in the console).
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: bare-metal/apple-silicon/quickstart.mdx [hunk 8]
```diff
@@ -51,6 +51,11 @@ You can access your Mac mini using VNC, a remote desktop-sharing protocol. It al
 
 Refer to our detailed documentation for Windows, Linux, and macOS for OS specific information: [How to access the remote desktop of a Mac mini](/bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini/).
 
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
+  For manual connections, ensure that the port from the "Overview" section is correctly entered in your VNC client settings, as this is necessary for a successful connection.
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/quickstart.mdx:56]:
```suggestion
  For manual connections, ensure that the port from the **Overview** section is correctly entered in your VNC client settings, as this is necessary for a successful connection.
```

File: bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx [hunk 9]
```diff
@@ -47,13 +47,18 @@ round-trip min/avg/max/stddev = 0.182/0.207/0.261/0.028 ms
 
 ## 2. Verify the VNC connection
 
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the "Overview" section of your Mac mini’s configuration.
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx:51]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```

File: bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx [hunk 10]
```diff
@@ -47,13 +47,18 @@ round-trip min/avg/max/stddev = 0.182/0.207/0.261/0.028 ms
...
+  For manual connections, ensure that the port from the "Overview" section is correctly entered in your VNC client settings, as this is necessary for a successful connection.
```
Comment (jcirinosclwy) [inline bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx:52]:
```suggestion
  For manual connections, ensure that the port from the **Overview** section is correctly entered in your VNC client settings, as this is necessary for a successful connection.
```

**Review by jcirinosclwy — APPROVED** [review]

File: bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx [hunk 11]
```diff
@@ -26,6 +26,12 @@ The VNC protocol is platform-independent, supporting Linux, Windows, and macOS,
 - A [Mac mini](/bare-metal/apple-silicon/how-to/create-mac-mini/)
 
 ## Accessing the remote desktop
+
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```
Comment (ldecarvalho-doc) [inline bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini.mdx:31]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and may therefore be different from the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```

File: bare-metal/apple-silicon/quickstart.mdx [hunk 12]
```diff
@@ -51,6 +51,11 @@ You can access your Mac mini using VNC, a remote desktop-sharing protocol. It al
 
 Refer to our detailed documentation for Windows, Linux, and macOS for OS specific information: [How to access the remote desktop of a Mac mini](/bare-metal/apple-silicon/how-to/access-remote-desktop-mac-mini/).
 
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```
Comment (ldecarvalho-doc) [inline bare-metal/apple-silicon/quickstart.mdx:55]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and may therefore be different from the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```

File: bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx [hunk 13]
```diff
@@ -47,13 +47,18 @@ round-trip min/avg/max/stddev = 0.182/0.207/0.261/0.028 ms
 
 ## 2. Verify the VNC connection
 
+<Message type="important">
+  The VNC port on Mac mini systems is **randomly assigned** during system installation and does not use the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```
Comment (ldecarvalho-doc) [inline bare-metal/apple-silicon/troubleshooting/cant-connect-using-vnc.mdx:51]:
```suggestion
  The VNC port on Mac mini systems is **randomly assigned** during system installation and may therefore be different from the default VNC port 5900. To connect successfully via VNC, use the specific port shown in the **Overview** section of your Mac mini’s configuration.
```

**Pull request merged**